platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/TemporaryFileBackedOutputStream.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import com.google.common.io.ByteSource;
+import com.google.common.io.FileBackedOutputStream;
+
+/**
+ * TemporaryFileBackedOutputStream is a wrapper around {@link FileBackedOutputStream}. This class
```

Comment:
Since FBOS is now more of an "implementation detail", should we even mention it here? Maybe just explain how this class works and stop there. That way we can use a different class if we want in the future without being tied to FBOS.